skimage/filters/ridges.py

```diff
@@ -213,7 +213,11 @@ def meijering(image, sigmas=range(1, 10, 2), alpha=None,
             auxiliary = np.max(auxiliary, axis=0)
 
             # Rescale image intensity
-            filtered = np.abs(auxiliary) / np.abs(np.max(auxiliary))
+            denum = np.abs(np.max(auxiliary))
+            if denum == 0.:
+                filtered = np.zeros_like(auxiliary)
+            else:
+                filtered = np.abs(auxiliary) / denum
```

Comment:
Are both branches covered by a test? Also, denum -> denom.

But anyway, maybe I would rework this as:

```python
# use max(..., 1) to avoid division by 0
filtered = np.abs(auxiliary) / max(np.abs(np.max(auxiliary)), 1)
```

I also wonder whether the order of abs and max is correct... It's a bit weird to take the abs of the max rather than the max of the abs...

Reply:
I think auxiliary was a float

Reply:
what I mean is that `np.abs(np.max([-1, -2]))` is 1, while `np.abs(np.max([-1, -2, 0.5]))` is 0.5. The converse is 2 in both cases. I don't know which is correct in this case, but I've encountered `np.max(np.abs())` more in my life. =P

Reply:
I think almost always you rather want `max(abs(...))`.  Depends on what you want, of course, but the other version is likely to give you something close to zero for negative values—seldom what you want.

Reply:
@stefanv if it's something like curvature, I can imagine you want negative values to be "clipped" to be close to 0, but then you probably want `clip(max())` rather than `abs(max())`. Probably best to ask the original author. @DavidBreuer?